Gemfile.lock

```diff
@@ -735,7 +735,7 @@ GEM
     faraday_middleware (0.12.2)
       faraday (>= 0.7.4, < 1.0)
     fastimage (2.1.0)
-    ffi (1.9.17)
+    ffi (1.9.24)
```

Comment:
@jkassemi @kreek - Be aware, CI was reporting a failure due to a gem dependency issue. Here is the associated commit:  https://github.com/department-of-veterans-affairs/vets-api/pull/2255/commits/ede69d2552fc030c35287c7d2c23da66d18e44ca

Here is the page that is referenced in the commit:  https://github.com/ffi/ffi/releases/tag/1.9.24